packages/frontend/@n8n/design-system/src/components/N8nMenuItem/MenuItem.vue

```diff
@@ -83,7 +84,10 @@ const iconColor = computed(() => {
 					>
 					<N8nIcon v-else-if="icon" :color="iconColor" :icon="icon" />
 				</div>
-				<N8nText v-if="!compact" :class="$style.menuItemText">{{ item.label }}</N8nText>
+				<N8nText v-if="!compact" :class="$style.menuItemText" color="text-dark">
```

Comment:
Changed the color to darker black after syncing with Rob. New designs have `black 800` as the color, but since we haven't taken the new tokens into use yet the side menu looks currently a bit washed out.

Once we take the `_tokens.new.scss` into use this should go back to using `text-base`, but for now `text-dark` here results into `black 850` which is closer to what we want.